test/integration/concierge_credentialrequest_test.go

```diff
@@ -67,8 +67,15 @@ func TestSuccessfulCredentialRequest_Browser(t *testing.T) {
 			},
 		},
 		{
-			name:          "jwt authenticator",
-			authenticator: testlib.CreateTestJWTAuthenticatorForCLIUpstream,
+			name: "jwt authenticator",
```

Comment:
Copyright year is not updated in this file.

Reply:
Ah, before I added the hook. 👍